docs/setting-up/server/network.md

```diff
@@ -0,0 +1,22 @@
+# Network
+
+## Ports
+
+This is a list of ports used by the various components of PMM.
+
+For PMM to work correctly, your system's firewall should allow traffic on these ports.
+
+| Destination   | Component     | Protocols | Port/Port Range | Dicertion | Description
+|---------------|---------------|-----------|-----------------|-----------|---------------------
+| PMM Server    | `pmm-server`  | TCP       |   80            | out       | HTTP server, used for web interface
+| PMM Server    | `pmm-server`  | TCP       |  443            | out       | HTTPS server, used for web interface
```

Comment:
this is also how pmm-agent communicates to pmm-server so there's web traffic AND gRPC traffic over HTTPS